pkg/test/masteruserrecord/assertion.go

```diff
@@ -144,6 +144,7 @@ func (a *Assertion) AllUserAccountsHaveTier(tier toolchainv1alpha1.NSTemplateTie
 	for _, ua := range a.masterUserRecord.Spec.UserAccounts {
 		a.userAccountHasTier(ua, tier)
 	}
+	assert.Equal(a.t, tier.Name, a.masterUserRecord.Spec.TierName)
```

Comment:
wouldn't it be better to create a new function for this particular field? There are two reasons:
* it's not related to UserAccounts/NSTemplateSet CRDs
* we don't miss this verification code as soon as we drop NSTemplateSet from MUR

Reply:
right, I can do

Reply:
done in 1f17dba